pom.xml

```diff
@@ -140,17 +141,23 @@
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>emitter</artifactId>
-                <version>0.4.1</version>
+                <version>0.4.5</version>
             </dependency>
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>http-client</artifactId>
-                <version>1.0.6</version>
+                <version>1.1.0</version>
             </dependency>
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>server-metrics</artifactId>
                 <version>0.2.8</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.metamx</groupId>
+                        <artifactId>java-util</artifactId>
```

Comment:
It's to resolve conflict, I added this because some other library needs newer version of java-util than server-metrics depend on, maybe emitter or http-client